dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jsonpInternal-2.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jsonpInternal-2.0
+visibility=private
+-features=com.ibm.websphere.appserver.jsonpImpl-2.0.0; ibm.tolerates:=""
```

Comment:
why an empty tolerates clause here?

Reply:
I left that on purpose to indicate no tolerations.  It is vestigial and I've no problem removing the clause.

Reply:
Lets discuss this on our upcoming call, but I think we will still need to create a `jsonpImpl-2.0.1` feature

Reply:
Followed-up: This is not an issue.